base/collection_xattr.go

```diff
@@ -293,17 +293,18 @@ func (c *Collection) subdocGetBodyAndXattrs(ctx context.Context, k string, xattr
 
 		c.Bucket.waitForAvailKvOp()
 		defer c.Bucket.releaseKvOp()
-
 		// First, attempt to get the document and xattr in one shot.
 		ops := make([]gocb.LookupInSpec, 0, len(xattrKeys)+1)
 		for _, xattrKey := range xattrKeys {
+			if xattrKey == "" {
```

Comment:
Do we care that the mutateIn functions that iterate over xattrKeys don't have similar defensive checks for empty keys?  (it's fine if the answer is 'no we don't' - just wanted to get your thoughts on it).